### PR TITLE
AG-9950 Add inner padding to cross lines on band scale axes

### PR DESCRIPTION
**Pull request by lsjroberts**
https://ag-grid.atlassian.net/browse/AG-9950

Before
Screenshot 2023-11-28 at 16 28 28

After
Screenshot 2023-11-28 at 16 28 39

### CODE REVIEW

**Review by alantreadway — COMMENTED** [review]
Changed LGTM - guessing some snapshot updates are missing?

File: packages/ag-charts-community/src/chart/crossline/__image_snapshots__/cross-line-test-ts-cross-lines-create-for-area-crosslines-it-should-render-to-canvas-as-expected-1-snap.png
Comment (iMoses) [inline packages/ag-charts-community/src/chart/crossline/__image_snapshots__/cross-line-test-ts-cross-lines-create-for-area-crosslines-it-should-render-to-canvas-as-expected-1-snap.png:1]: that's the only snapshot which looks a bit weird IMO

Comment (alantreadway) [reply packages/ag-charts-community/src/chart/crossline/__image_snapshots__/cross-line-test-ts-cross-lines-create-for-area-crosslines-it-should-render-to-canvas-as-expected-1-snap.png:1]: Verified with @AG-DavidG that the new version is correct.

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**